src/session.rs

```diff
@@ -3235,6 +3256,15 @@ impl Session {
             Command::SelectionYank => {
                 self.yank_selection();
             }
+            Command::SelectionFlip(dir) => {
+                self.flip_selection(dir);
+                // I think its handy to flip in place for now, hence these commands
+                // 1., 2. prevent overlap and paste
+                // 3.     preserve location so you can flip repeatedly w/ hotkey
+                self.command(Command::SelectionErase);
+                self.command(Command::SelectionPaste);
+                self.command(Command::Mode(Mode::Visual(VisualState::Selecting { dragging: false })));
+            }
```

Comment:
We should check that there is an active selection first, and do nothing if there isn't. Also check that we're in visual mode, but not pasting (ie. not with something yanked already)

Reply:
Alright, will do!

Reply:
this should also be resolved, if I understand it, I think?